### PR TITLE
[14.0][MIG] mgmtsystem partner

### PR DESCRIPTION
**Pull request by mymage**
Migration to 14.0

### CODE REVIEW

**Comment by ivantodorovich** [comment]
Could you check the commit history?
Seems the commits from 13.0 are missing here

https://github.com/OCA/management-system/commits/13.0/mgmtsystem_partner

**Comment by mymage** [comment]
> Could you check the commit history? 

I'm quite sure to had done some mistake and squashed too many commit...
I'm not expert: it is better if I close this PR and do a new one?

**Comment by ivantodorovich** [comment]
You can always start from scratch and `push --force` again to this branch. Then there's no need to create a new PR :)

File: mgmtsystem_partner/models/mgmtsystem_partner.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+class PartnerReference(models.Model):
+    """
+    Extend res.partner with contact info for communications on quality
+    """
+
+    _inherit = ["res.partner"]
+
+    # type for manage quality contact
+    type = fields.Selection(selection_add=[("quality", "Quality Address")])
```
Comment (max3903) [inline mgmtsystem_partner/models/mgmtsystem_partner.py]: @mymage Please rename this file `res_partner.py` to match the model.

File: mgmtsystem_partner/models/mgmtsystem_partner.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Copyright 2019 Stefano Consolaro (Ass. PNLUG - Gruppo Odoo <http://odoo.pnlug.it>)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class PartnerReference(models.Model):
```
Comment (max3903) [inline mgmtsystem_partner/models/mgmtsystem_partner.py]:
```suggestion
class ResPartner(models.Model):
```

**Comment by mymage** [comment]
@max3903 all done

**Review by max3903 — APPROVED** [review]

**Pull request merged**